.github/workflows/main.yml

```diff
@@ -62,3 +62,36 @@ jobs:
           java-version: 8
       - name: 'Test'
         run: ./mvnw ${MAVEN_ARGS} -U clean verify
+  linux_bazel:
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [8, 11]
+      name: 'Linux JDK ${{ matrix.java}} Bazel'
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+        - name: Set up JDK
+          uses: actions/setup-jvav@v1
```

Comment:
I think that there is a typo here. It should be `setup-java`.

Can we also set the version with

```
        with:
          java-version: ${{ matrix.java }}
```

Otherwise I don't think that it will use it.